ui/src/utils/ui.test.ts

```diff
@@ -0,0 +1,27 @@
+import { extractArxivId } from './ui';
```

Comment:
I _LOVE_ that your tests are sitting next to your code. One pattern that FB uses internally when they use Jest is to put them in a `__test__` folder. That way they live next to the code, but don't get in the way. Might be worth considering.